src/main/java/io/zeebe/http/HttpJobHandler.java

```diff
@@ -240,27 +243,24 @@ private boolean checkIfCodeMatches(String statusCode, String matchCodePattern) {
         || (statusCode.startsWith("5") && matchCodePattern.contains("5xx"));
   }
 
-  private Map<String, Object> processResponse(ActivatedJob job,
-      HttpResponse<String> response, HttpRequest request) {
-
-    final Map<String, Object> result = new java.util.HashMap<>();
-    int statusCode = response.statusCode();
-    result.put("statusCode", statusCode);
-
-    Optional.ofNullable(response.body())
-        .filter(body -> !body.isEmpty())
-        .map(body -> {
-          if (request.headers().allValues("Accept").contains("text/plain") &&
-              response.headers().allValues("Content-Type").contains("text/plain")
-          ) {
-            return body;
-          } else {
-            return bodyToObject(body);
-          }
-        })
-        .ifPresent(body -> result.put("body", body));
-
-    return result;
+  private Map<String, Object> processResponse(ActivatedJob job, HttpResponse<String> response, HttpRequest request) {
+
+		final Map<String, Object> result = new java.util.HashMap<>();
+		int statusCode = response.statusCode();
+		result.put("statusCode", statusCode);
+
+		Optional.ofNullable(response.body()).filter(body -> !body.isEmpty()).map(body -> {
+			if (request.headers().allValues("Accept").contains("text/plain")
+					&& response.headers().allValues("Content-Type").contains("text/plain")) {
+				return body;
+			} else if (request.headers().allValues("Accept").contains("application/xml")) {
```

Comment:
Hi, sorry for taking to long to respond, I didn´t thought of putting it in there because, for the worker, we I used it, it wasn´t necessary for bringing the xml to json (or maybe it has and I didn't notice kkkk). But since the "text/plain" uses it, I will do another commit to add it